cmd/ipfs-cluster-ctl/main.go

```diff
@@ -391,29 +395,36 @@ If you prefer faster adding, add directly to the local IPFS and trigger a
 				wg.Add(1)
 				go func() {
 					defer wg.Done()
-					var last *api.AddedOutput
+
+					var buffered []addedOutputQuiet
+					var lastBuf = make([]addedOutputQuiet, 1, 1)
+					var qq = c.Bool("quieter")
+					var q = c.Bool("quiet") || qq
```

Comment:
On running with `quiet` and `quieter`, it is also printing name and size.

Reply:
should be fixed to after `--no-stream`